Frontend/src/app/modules/profiles/profiles.component.html

```diff
@@ -83,23 +83,26 @@ <h1 appTitle class="cdk-visually-hidden">{{ 'PROFILES.TITLE' | translate: {name:
       </div>
       <div class="statsBackground">
         <div class="stats">
-          <div>
+          <div [class.stat-link]="canActivateLinks()"
+               (click)="canActivateLinks() && navigateTo('workspace')" >
             <div class="stat">
               {{'PROFILES.STATS.MATERIALS' | translate}}
             </div>
             <div class="value">
               {{ userStats.nodeCount }}
             </div>
           </div>
-          <div>
+          <div [class.stat-link]="canActivateLinks()"
```

Comment:
Try to use "routerLink" if possible, since this will support default browser behaviour for links and improves accessibillity